docs/DeveloperGuide.md

```diff
@@ -1,13 +1,286 @@
+If the newly created `Author` is undocumented by `Tantou`, a `TantouException` is thrown, informing the user that this `Author` does not exist and the `Manga` cannot be removed.
+If the `Author` instead exists, `Tantou` will check for the existence of the newly created `Manga`. If there is no existing
+association between the `Manga` and `Author`, a `TantouException` is thrown, informing the user that they are deleting a non-existing `Manga`. Otherwise,
+the `Manga` is removed from the `Author`'s `MangaList` and the current state of `AuthorList` is saved via `Storage` for
+data persistency.
+#### Interaction
+The following diagram illustrates the interactions that take place when the
+user provides `"catalog -a Kubo Tite -m Bleach -d"` as an input.
+![add manga sequence diagram](/docs/uml/images/DeleteMangaSequence.png)
+
+### View Command
+#### Overview
+The `ViewAuthorsCommand` and `ViewMangasCommand` are responsible for displaying a list of the various data entries in `AuthorList`. Using the [`Ui` class](#displaying-data), it formats the data into a table. 
+
+For example, `view -a test1 -b -s` gives the following output (`b` for by-date/deadline, `s` for sales data). 
+```
+view -a test1 -b -s
+Mangas authored by test1, Total: 2
+no. | Manga Name                               | Deadline             | Unit Price | Units Sold | Revenue
+-----------------------------------------------------------------------------------------------------------------
+  1 | manga 1-1                                | None                 | N/A        | N/A        | N/A
+  2 | manga 1-2                                | None                 | N/A        | N/A        | N/A
+```
+
+
+<h4 id="view-command-interaction">
+Interaction
+</h4>
+
+The following UML sequence diagrams illustrate the interactions that take place when the user provides a valid `ViewMangasCommand` command (e.g. `view -a test1 -b -s`, where `test1` is an author that already wrote some manga). 
+![ViewMangaSequence.png](uml/puml/ViewMangaSequence/ViewMangaSequence.png)
+![refGetColumnsToPrintSequence.png](uml/puml/ViewMangaSequence/refGetColumnsToPrintSequence.png)
+![refPrintRowsSequence.png](uml/puml/ViewMangaSequence/refPrintRowsSequence.png)
+`ViewAuthorsCommand` works similarly, but with only 2 required columns to print (row number and author name). 
+
+### AddSalesCommand
+#### Overview
+The AddSalesCommand is responsible for adding sales data to a Manga. The Sale data consists of three attributes: `quantitySold`, `unitPrice` and `totalRevenue`. The `quantitySold` and `unitPrice` are inputs from the user, while `totalRevenue` is calculated by multiplying y`quantitySold` and `unitPrice`.
+
+For the AddSalesCommand to be successful, the manga that the sales data is associated with must exist. If the `sales`
+command is successful, the `Sales` data is then saved via Storage.
+
+![mangasales_class.png](uml/images/mangasales_class.png)
```

Comment:
I think this class diagram is not relevant to this section and shouldn't be placed here?